charts/crljanitor/values.yaml

```diff
@@ -29,6 +29,9 @@ vault:
   pathPrefix:
   # vault.configPathPrefix -- (string) Vault path prefix for configs. Required if vault.enabled.
   configPathPrefix:
+  # vault.adminUserFilePath -- (string) Vault path prefix for admin user list. Required if vault.enabled.
+  # Use the same users as admin for all env by default. Override if needed in helmfile repo.
+  adminUserFilePath: config/terra/crl-janitor/common/iam
```

Comment:
I'm a little confused about the nature of this chart value. Is this path a Vault path, or a file path accessed from Java within the Docker container? The docs suggest the latter, but I think it's more the former.

Possible var names which might be clearer:
- adminUserSecretPath
- adminUserList
- adminUserListSecret

Reply:
Yes, this is Vault path. And the path is , in that path, there would be bunch of key value paris inside. I plan to put all future potential IAM realated configs inside. So I name it as `config/terra/crl-janitor/common/iam`. It would have adminUser, SAM related, IC related things. 
Currently, inside `config/terra/crl-janitor/common/iam` path, there is a key adminUsers and the value is our config.